ci/long_running_tests/README.rst

```diff
@@ -6,45 +6,39 @@ forever until they fail. To set up the project you need to run
 
 .. code-block:: bash
 
-    pip install anyscale
-    anyscale project create
+    $ pip install anyscale
+    $ anyscale init
 
 Note that all the long running test is running inside virtual environment, tensorflow_p36
 
 Running the Workloads
 ---------------------
+Easiest approach is to use the `Anyscale UI <https://www.anyscale.dev/>`. First run ``anyscale snapshot create`` from the command line to create a project snapshot. Then from the UI, you can launch an individual session and execute the run command for each test. 
 
-You can start all the workloads with:
+You can also start the workloads using the CLI with:
 
 .. code-block:: bash
 
-    anyscale session start -y run --workload="*" --wheel=https://s3-us-west-2.amazonaws.com/ray-wheels/releases/0.7.5/6da7eff4b20340f92d3fe1160df35caa68922a97/ray-0.7.5-cp36-cp36m-manylinux1_x86_64.whl
+    $ anyscale start
+    $ anyscale run run --workload=<WORKLOAD_NAME> --wheel=<RAY_WHEEL_LINK>
 
-This will start one EC2 instance per workload and will start the workloads
-running (one per instance). You can start a specific workload by specifying
-its name as an argument ``--workload=`` instead of ``"*"``. A list of
-available options is available via `any session start run --help`.
 
+Doing this for each workload will start one EC2 instance per workload and will start the workloads
+running (one per instance). A list of
+available workload options is available in the `ray_projects/project.yaml` file.
 
-Check Workload Statuses
------------------------
-
-To check up on the workloads, run either
-``anyscale session --name="*" execute check-load``, which
-will print the load on each machine, or
-``anyscale session --name="*" execute show-output``, which
-will print the tail of the output for each workload.
```

Comment:
any reason to delete this?

Reply:
@rkooo567 can confirm, but I don't think these commands were actually used for debugging during the release process. Plus I don't think they work as intended because the command output is not piped to the terminal and `test_workload` doesn't actually get run in tmux on the product. I left these as comments for now in case we want to add them back once the issues on the product get fixed